web-apis.md

```diff
@@ -0,0 +1,97 @@
+# Web APIs
+
+## Introduction to Web APIs
+
+Web APIs are interfaces provided by the browser that allow you to interact with the browser and the underlying operating system.
+
+## DOM Manipulation
+
+The Document Object Model (DOM) is a programming interface for web documents. It represents the page so that programs can change the document structure, style, and content.
+
+### Example Code: Changing the Content of an Element
+
+```document.getElementById("demo").innerHTML = "Hello, World!";```
+
+##Fetch API 
+
+The Fetch API provides an interface for fetching resources (including across the network).
+
+###Example Code: Simple Fetch Request
+
+```
+fetch('https://api.example.com/data')
+  .then(response => response.json())
+  .then(data => console.log(data))
+  .catch(error => console.error('Error:'));```
+
+##Geolocation API
+
+The Geolocation API allows the user to provide their location to web applications if they so desire. For privacy reasons, the user is asked for permission to report location information.
+
+###Example Code: Getting User Location
+
+```
+if ("geolocation" in navigator) {
+  navigator.geolocation.getCurrentPosition(function(position) {
+    console.log("Latitude: " + position.coords.latitude);
+    console.log("Longitude: " + position.coords.longitude);
+  });
+} else {
+  console.log("Geolocation is not supported by this browser.");
+}```
+
+##Canvas API
+
+The Canvas API provides a means for drawing graphics via JavaScript and the HTML <canvas> element.
+
+###Example Code: Drawing Shapes on Canvas
+
+```
+var canvas = document.getElementById('myCanvas');
```

Comment:
Would be nice to use `let`  or `var` in all place.

Reply:
Also, the markdown syntax is not correct here. I suggest looking at other chapters and previewing your work before pushing it.

Reply:
I have done the changes required. Do I have to mention the chapter name and page number as mentioned by other authors.

Reply:
Thanks for making changes. I see some merge conflicts. Can you fix it? 

Regarding the chapter and page number. You can leave it to me.

Reply:
Sure, I will try.